### PR TITLE
this PR contains only deletions

### PR DESCRIPTION
**Pull request by AdamElTablawy**
AAAAAAAAAARGH FUCK YOUFUCKYOUFUCK YOUF CUK YOU FUCK YOU

FUCK THIS COMBAT MODE BULLSHIT

FUCK YOU

GONE

SPRINTING IS GONE

COMBAT MODE RANDOMNESS/MECHANICAL DOWNSIDES FOR HAVING IT ON IS GONE

I FIXED PA HELMETS HAVING DEFLECTION LIKE POTS ASKED

FUUUUUUUUUUCK YOU

FUCK YOU

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

:cl:
del: Sprinting, several combat mode mechanical effects (such as combat mode affecting punches, et cetera), staminacombat effects (such as punching draining your stamina and batonning someone draining your stamina), power armor helmets having deflection, and unused defines
/:cl:

### CODE REVIEW

File: code/game/objects/items/stunbaton.dm [hunk 1]
```diff
@@ -155,18 +155,11 @@
...
 	user.DelayNextAction()
 	if(ishuman(M))
 		var/mob/living/carbon/human/L = M
 		if(check_martial_counter(L, user))
 			return TRUE
-	if(turned_on)
-		if(baton_stun(M, user, disarming))
-			user.do_attack_animation(M)
-			user.adjustStaminaLossBuffered(getweight(user, STAM_COST_BATON_MOB_MULT))
 	else if(user.a_intent != INTENT_HARM)			//they'll try to bash in the last proc.
```
Comment (buffyuwu) [inline code/game/objects/items/stunbaton.dm:163]:
```suggestion
	if(user.a_intent != INTENT_HARM)			//they'll try to bash in the last proc.
```

**Comment by buffyuwu** [comment]
ok uhhhh


why


why are you removing stamina entirely

**Comment by AdamElTablawy** [comment]
> ok uhhhh
> 
> why
> 
> why are you removing stamina entirely
> 
> edit: worthy note there are.. many errors in how you're going about this as well including space indentation

i have noticed the space indentation and am tearing it out

i'm not removing stamina entirely but i am removing a lot of stamina combat because it is:

bad.

stamina combat is bad, basically. pumping shotguns draining stamina is bad. punching people draining stamina is bad. staminacombat in general wasn't designed for f13 and it doesn't really work for f13 overall. for instance, getting exhausted while punching people.  i tore  out melee draining stamina a while ago because it made melee useless as compared to guns but the same problem continues to exist with punching et cetera.

we also had staminacombat effects like struggling to inflict proper damage or getting aim inaccuracy while fighting because you clicked too rapidly-a lot of it is citcode holdovers that add unseen/unintended combat buffs/strange effects that we don't really need. 

a full list: stamcrit item penalties (this is basically unused, ever since i removed items draining stamina on hit, but we don't really want or need that feature (otherwise it makes guns always superior to melee weapons) 

stamcrit item attack delays (ditto)

combat mode modifications to unarmed damage (we want combat mode to basically exist for the identifier only, without mechanical effects)

eyestabbing (again, yet more melee disadvantages for no reason based off of stamina combat)

baton stamina (again, this is a holdover from baton-centric ss13. here batons are really only used by MPs and we don't want to make their jobs harder when arresting troopers, especially since troopers can just shoot them to death faster)

stamina drains when throwing objects, people, items (again, this is an unnecessary weakness provided to everything but guns. we don't really want to make people suffer stamina drains and weaknesses for choosing to use throwing spears, melee, so on and so forth, lest we force people to ONLY use guns-a consistent complaint is that melee is useless, and one major way to avoid this is to cut out all of the little things that make guns mechanically superior, and one of those was the fact that guns were immune to the 'drain stamina' system

disarming in combat mode (we again want combat mode to be purely identifier, not mechanical, and having unseen changes to it is a citadel holdover)

stamina regeneration in combat mode (again, we don't want to punish people for using combat mode which is required per rules)

aiming delay (in practice, this actually really only affected snipers and launchers, burst weapons did not suffer from this)

stamina on burstfire weapons (this didn't actually really do anything in game-the gun 'recoil' draining stamina did not function, and you could fire a bozar for a solid hour without suffering stamina loss


stamina as a damage type and as inflicted by batons/flips/whatever still remains, but staminacombat as a system is a citadel holdover and a mostly unnecessary one

**Comment by buffyuwu** [comment]
a lot of this is misunderstood, like for example thrown weapons cost stamina but are meant to be hilariously effective. that's the cost. if the f13 spears dont have the damage to award the loss, that's the spears needing a buff not the stamina system needing removal.

if you want to see improvements to the combat system, deleting it is not the way to go. port more of it down from upstream or write a new one, but do not just blindly delete code because you disagree with it please

**Pull request closed without merging**